packages/itwinui-react/src/core/DatePicker/DatePicker.tsx

```diff
@@ -594,21 +639,27 @@ export const DatePicker = React.forwardRef((props, forwardedRef) => {
         <div
           onKeyDown={handleCalendarKeyDown}
           role='listbox'
+          {...calendarProps}
+          className={cx(className, calendarProps?.className)}
```

Comment:
since there is nothing to merge here, it doesn't even need to be set explicitly. `{...calendarProps}` will take care of it
```diff
- className={calendarProps?.className}
```